uptane-whitepaper.md

```diff
@@ -1 +1,106 @@
+
+Unfortunately,connecting ECUs to the Internet make them vulnerable to a wide range of attacks, some of which, like SolarWinds, introduce malware masquerading as legitimate updates. The potential consequences of such attacks could be costly indeed, not only in terms of recalls or lost sales, but also potentially, in loss of life.
+
+The types of attacks an automotive software security system needs to to defend against can be organized into four categories, presented here in order of increasing severity.
+
+**Read updates:** The goal here is intellectual property theft, so these attackers aim to read the contents of software updates. This is generally achieved with an Eavesdrop attack, where attackers can read unencrypted updates sent from the repository to the vehicles.
+
+**Deny updates:** In this group of attacks, the goal is to deny access to updates so vehicles cannot fix software problems, including newly discovered vulnerabilities. These strategies include:
+- *Drop-request attack:* blocks network traffic outside or inside the vehicle to prevent an ECU from receiving any updates.
+- *Slow retrieval attack:* slows delivery time of updates to ECUs so a known security vulnerability can be exploited before a corrective patch is received.
+- *Freeze attack:* continues to send the last known update to an ECU, even if a newer update exists.
+- *Partial bundle installation attack:* Allows only part of an update to install by dropping traffic to selected ECUs.
+
+**Deny functionality:** This grouping ups the threat ante a bit further by causing vehicles to fail to function in one of the following ways:
+- *Rollback attack:* tricks an ECU into installing outdated software with known vulnerabilities.
+- *Endless data attack:* causes an ECU to crash by sending it an infinite amount of data until it runs out of storage.
+- *Mixed-bundles attack:* shuts down an ECU by causing it to install incompatible versions of software updates that must not be installed at the same time. Attackers can accomplish this by showing different bundles to different ECUs at the same time.
+- *Mix-and-match attack:*  causes an ECU to use arbitrary combinations of new versions of images.
```

Comment:
What's the difference between "mixed bundles" and "mix-and-match"? Sounds like the same thing to me.

Reply:
I went back to the original escar paper and based on how the attack is described there, I propose the following clarification. Does this make the difference clearer?

```suggestion
- *Mixed-bundles attack:* shuts down an ECU by causing it to install incompatible versions of software updates that must not be installed at the same time. Attackers can accomplish this by showing different bundles to different ECUs at the same time.
- *Mix-and-match attack:*  Like the mixed-bundles attack described above, this attack also causes ECUs
 to use arbitrary combinations of new versions of images. However, it is a more serious threat as the ability to initiate it means attackers have abused repository keys to sign this bundle, and the updates provided can be completely arbitrary.

Reply:
Yep, that makes more sense, thanks! There's an unnecessary linebreak in there, but that's not a big deal.